README.md

```diff
@@ -10,13 +10,64 @@ algebraic systems. Additional modules support input and output as well
 as basic profiling and error management.  
 
 <!---See the [GridPACK home page](https://www.gridpack.org) for more information.-->
+The GridPACK framework will simplify parallel power grid application development
+by eliminating many of the index transformations and calculations required to
+distribute data in parallel applications and also to move data from one
+processor to another as the calculation proceeds. Application writers can focus
+primarily on the physics of their problem without worrying about much of the
+baggage associated with writing applications in parallel. GridPACK supplies core
+functionalities that will simplify parallel code development. These include a
+generic framework for setting up power grid networks and applying whatever
+physical properties to the bus and branches that developers want, functionality
+for distributing the networks across processors and managing the exchange of
+network data between processors, generic functionality for mapping data from the
+networks to matrices and vectors, and libraries of parallel matrix and vector
+operations, linear and non-linear solvers and preconditioners for solving the
+equations generated by power grid models. The framework will also supply
+routines for generating networks from standard format input files that describe
+power grid networks (e.g. PTI files), writing data from the network back out to
+files, and additional functionality for profiling and error management. The
+modules in the framework are being designed to compartmentalize functionality in
+a way that will simplify future code maintenance. This compartmentalization will
+also allow modules to be updated more easily without requiring modification of
+other modules or of the applications.
 
-## Installing
+If you have any difficulties building or installing GridPACK, or have any
+questions, feel free to contact us at
+[gridpack.account@pnnl.gov](mailto:gridpack.account@pnnl.gov). We look forward
+to hearing from you. You can also post issues to the Github issue page for the
+GridPACK repository, located
+[here](https://github.com/GridOPTICS/GridPACK/issues).
 
-GridPACK can be built in a number of different ways. The `docs/notes` subdirectory has instructions for building GridPACK on different clusters. Also, it includes a [step-by-step guide](docs/notes/install.md) for building, installing GridPACK and its dependencies.
+## User Information
+
+- [Installation](docs/markdown/BASIC_INSTALL.md)
+- [GridPACK User Manual](docs/user_manual/GridPACK.pdf) (We recommend you download the GridPACK.pdf file to your computer)
+- [Downloads](https://github.com/GridOPTICS/GridPACK/releases)
+- [FAQS](docs/markdown/FAQS.md)
+- [Tutorials](docs/markdown/TUTORIALS.md)
+- [Data Sets](docs/markdown/DATASETS.md)
+- [License](docs/markdown/LICENSE.md)
+<!--
+- [Building GridPACK from Scratch](docs/markdown/BUILD_GRIDPACK.md)
+-->
+
+## Submodule(s)
```

Comment:
Submodules can be moved to installation

Reply:
I moved this text to the start of the basic install document. This information is also available in a few other places.